### ISSUE TITLE
Audio thumbnail does not have a descriptive alt text

### ISSUE THREAD

**Issue by obulat** [WordPress/openverse-frontend#1854]
## Problem

Alt attribute on the audio thumbnail image does not describe the image.

## Description

A descriptive alt attribute should be provided when possible.

## Additional context

Reported by Joe Dolson (https://make.wordpress.org/openverse/2022/09/07/frontend-release-v3-4-8-and-a-call-for-a11y-testing/#comment-95)

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

**Comment by dhruvkb** [comment]
Does the audio thumbnail have any inherent value for someone using a screen-reader? Rather than the alt text being "Thumbnail for track _<insert track name here>_" (which will almost always be read in close proximity to the track name, I would much rather that the image was ARIA-hidden entirely.

Tagging @panchovm for UX guidance.

**Comment by zackkrida** [comment]
Looks like SoundCloud uses the track name as the alt text 😢

**Comment by obulat** [comment]
Ideally, we would have the descriptive texts for the thumbnails. But since we don't, I agree that aria-hiding the thumbnail element is a better solution.
Would we add `aria-hidden="true"` to the `div` that wraps both the fallback and the image here (instead of adding the `title`):
https://github.com/WordPress/openverse-frontend/blob/940b686b387e3fcc86a32dd6fdab15a60cc1b19b/src/components/VAudioThumbnail/VAudioThumbnail.vue#L3

**Comment by dhruvkb** [comment]
@obulat that seems ideal. By doing that, to an SR, it would seem as if there is no thumbnail.
